frontend-react/src/pages/error/ErrorPage.tsx

```diff
@@ -1,50 +1,52 @@
-// @ts-nocheck // TODO: fix types in this file
 import { Alert } from "@trussworks/react-uswds";
 import React from "react";
 import Helmet from "react-helmet";
 
 import { NotFound } from "./NotFound";
 import { UnsupportedBrowser } from "./UnsupportedBrowser";
 
-interface ErrorPageProps {
-    code?: string;
-    error?: string;
-    errorInfo?: React.ErrorInfo;
-    type?: "page" | "message";
-}
-
 /* INFO
    For consistency, when passing the code prop, please use these values
    e.g. <ErrorPage code={CODES.NOT_FOUND_404} />
  */
 export enum CODES {
     UNSUPPORTED_BROWSER = "unsupported-browser",
     NOT_FOUND_404 = "not-found",
+    UNKNOWN = "unknown-error",
+}
+
+interface ErrorPageProps {
+    code?: CODES.UNSUPPORTED_BROWSER | CODES.NOT_FOUND_404 | CODES.UNKNOWN;
```

Comment:
@kevinhaube This limits the possible values to a subset of values from CODES... but I wonder if I shouldn't just use the enum as a type?  `code?: CODES` ? (LOL.. code reviewing my own stuff.)

Reply:
Fixed

Reply:
I saw this and thought "man, it sucks we'd have to update both this and the enum. I had no idea you could use an enum as a type, though! My brain thought about whether we should make a type, but ultimately we would've just been changing where we updated the list of values from here to that type. The enum typing is great!